plugins/Deflect/DeflectPlugin.cpp

```diff
@@ -449,3 +458,12 @@ void DeflectPlugin::postRender()
     _impl->postRender();
 }
 }
+
+extern "C" brayns::ExtensionPlugin* brayns_plugin_create(const int argc,
+                                                         const char** argv)
+{
+    auto pm = brayns::DeflectParameters::createPropertyMap();
+    if (!pm.parse(argc, argv))
+        return nullptr;
+    return new brayns::DeflectPlugin(pm);
```

Comment:
Better:

    brayns::DeflectParameters params;
    if (!params.parse(argc, argv))
        return nullptr;
    return new brayns::DeflectPlugin(std::move(params));

That way createPropertyMap can become an implementation function and DeflectPlugin takes directly what it needs, which is the parameters.